package-res/ccc/core/base/chart/chart.js

```diff
@@ -208,6 +229,8 @@ def
         /* Increment create version to allow for cache invalidation  */
         this._createVersion++;
 
+        if(this.isCreated) 
```

Comment:
This is not needed. Or, at least indent the next line to be "inside" the new if :-)